sentry/src/main/java/io/sentry/SentryOptions.java

```diff
@@ -354,6 +354,31 @@ public class SentryOptions {
   /** ClientReportRecorder to track count of lost events / transactions / ... * */
   @NotNull IClientReportRecorder clientReportRecorder = new ClientReportRecorder(this);
 
+  /**
+   * Controls how many seconds to wait for sending events in case there were Startup Crashes in the
+   * previous run. Sentry SDKs normally send events from a background queue, but in the case of
+   * Startup Crashes, it blocks the execution of the {@link Sentry#init()} function for the amount
+   * of startupCrashFlushTimeoutMillis to make sure the events make it to Sentry.
+   *
+   * When the timeout is reached, the execution will continue on background.
+   *
+   * Default is 5000 = 5s.
+   */
+  private long startupCrashFlushTimeoutMillis = 5000; // 5s
+
+  /**
+   * Controls the threshold after the application startup time, within which a crash should happen
+   * to be considered a Startup Crash.
+   *
+   * Startup Crashes are sent on @{@link Sentry#init()} in a blocking way, controlled by
+   * {@link SentryOptions#startupCrashFlushTimeoutMillis}.
+   *
+   * Default is 2000 = 2s.
+   */
+  private long startupCrashDurationThresholdMillis = 2000; // 2s
```

Comment:
`m`: We didn't add this to the options on Cocoa. Do you expect many users need to tweak those settings?

Reply:
I don't, but I think we should definitely give them this option, 'cause startup times varies from app to app, and 2s might be too much or too little for some or the other. 

Did you expose the `startupCrashFlushTimeoutMillis` to the users though? That one I think we should definitely make configurable, because it's essentially blocking the app

Reply:
yea, but for how long, that's the question :) we chose reasonable defaults, but I think it should be configurable. It won't hurt I guess?

Anyway, I can also remove it, but I don't see it a problem if it's there.

Reply:
I try to avoid adding new options as much as I can. If you add it once, you have to support it. Adding it later is easy.

Reply:
I think I would at least keep the flushTimeout exposed since it controls for (up to) how long we are actually blocking the app of the users. Let's maybe ask what others think

Reply:
As discussed internally we'll add a kill-switch boolean flag for this feature